tests/invalid_models_tests/test_relative_fields.py

```diff
@@ -559,9 +559,12 @@ def test_related_field_has_invalid_related_name(self):
             'contains_%s_whitespace' % whitespace,
             'ends_with_with_illegal_non_alphanumeric_%s' % illegal_non_alphanumeric,
             'ends_with_whitespace_%s' % whitespace,
-            # Python's keyword
-            'with',
+            'with', # Python's keyword
+            'related_name\n',
         ]
+        from django.utils import six
```

Comment:
There are various reasons for doing this for example to mitigate design faults like circular dependencies or to postpone the construction triggers that could happen at compile time (i.e. when a class type is built) or even to keep the module namespace clear. Not sure without an example.